aws/logs_monitoring/lambda_function.py

```diff
@@ -464,5 +630,12 @@ def parse_service_arn(source, key, bucket, context):
             filesplit = filename.split("_")
             if len(filesplit) == 6:
                 clustername = filesplit[3]
-                return "arn:aws:redshift:" + region + ":" + accountID + ":cluster:" + clustername
+                return (
```

Comment:
prefer `"arn:aws:redshift:%s:%s:%s" % (region, accountId, clustername)`

Reply:
sounds good to me, I did not really touch this logic, it's just been reformatted by my linter